### DISCUSSION TITLE
심화 프로젝트 6주차 팀회고

### DISCUSSION THREAD

**Discussion by doinnn** [codeit-bootcamp-frontend/1-Weekly-Mission#568] [회고]
# 왜 회고를 해야 하나요?
바쁘고 치열하게, 또 잘 버티고 있는 스프린트 1기 여러분!
정신없이 프로젝트를 하다 보면 현재 우리 팀의 방향성과 목표를 잊어버리기 쉬워요.
매주 팀원들과 일주일을 회고하며 프로젝트를 위한 원동력이 되었으면 좋겠어요.

# 회고에 대한 궁금증
회고와 일기의 차이는 무엇인가요?
- 일기는 일상에서 있었던 일들을 의식의 흐름대로 편하게 적는 것이라면,
회고는 함께 보고 듣고 경험한 사건과 현상들을 곱씹어 보고, 그중에 특히 되짚고 싶은 것들을 **'왜' 와 '어떻게'에 집중해서 기록**하는 것입니다.

# 회고 방법
<img width="636" alt="스크린샷 2024-01-17 오전 10 43 31" src="https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/assets/152246380/c1e8d800-0ef4-4f68-9b6c-076de80d7c3d">

회고는 **“KPT 회고”** 방식으로 진행해볼거예요. 아래 방법대로 회고해주시면 됩니다.

* 팀원들과 모여 그 주의 팀 프로젝트 활동을 돌아보며, '좋았던 점(Keep) / 아쉬웠던 점(Problem) / 시도해볼 것(Try)'으로 항목을 나누어 회고해주세요.
   - Keep : 현재 우리팀이 잘 하고 있어서 계속 유지하면 좋을 것 같은 부분
   - Problem : 현재 우리팀이 가지고 있는 문제
   - Try : Problem에 적은 것을 해결하기 위해 우리 팀이 다음에 시도해볼만한 것 (혹은 Keep 중에서도 더 개선하거나 시도해보고 싶은 것)
* [KPT 회고 피그마 템플릿](<https://www.figma.com/file/qjYHQFGLwPcG7eWfVN7dEE/%ED%9A%8C%EA%B3%A0-%ED%85%9C%ED%94%8C%EB%A6%BF_1%EA%B8%B0?type=whiteboard&node-id=0-1&t=jp6YMOCJvuXCw68E-0>)을 활용해서, 팀 회고를 진행하셔도 됩니다. 
   - 단 템플릿 사용시, 팀별로 복제해서 사용해주세요. 꼭 템플릿을 사용하지 않아도 되니, 팀별로 편한 방식으로 회고를 진행해주세요.
* **다만 이번주 토요일부터 프로젝트가 시작되기 때문에 2주차부터 진행할게요. 다음주 일요일까지, 팀 회고한 내용을 정리하여 댓글로 올려주세요.** 
(댓글에 예시 회고를 올려드릴테니, 참고해주세요. 피그마 템플릿으로 진행하셨다면 캡쳐본을 올려주셔도 돼요.)

## 참고글

- [KPT 회고란 무엇인가?](<https://brunch.co.kr/@jinha0802/35>)
- [팀문화의 탄생](<https://techblog.woowahan.com/2677/>)

**Comment by doinnn** [comment]
# 🍀 0팀 (예시)

**Keep**
- 미팅 시간 약속을 모두 잘 지켜준다.
- 프로젝트 과정에서 기술 고민을 함께 하는 것이 서로의 학습에 도움이 된다.
- 새로운 제안을 했을 때, 모두 개방적으로 의견을 들어주어서 고맙다.
- 위워크에 모여서 하다보니, 프로젝트 진행이 빠르게 잘 이루어진다.

**Problem**
- 커밋 메시지가 명시적이지 않아서, 서로 어떤 작업을 했는지 파악하기 힘들었다.
- 급하게 코드를 작성하느라, 폴더 구조가 이상하다.
- 서로를 너무 배려하느라, 의견을 더 많이 나누지 못한다.

**Try**
- 협업을 위해, 커밋 메시지를 더 구체적으로 작성한다. (커밋 메시지 규칙 정하기)
- 폴더 구조를 다시 논의하여 폴더 구조 규칙을 만들고, 규칙에 맞추어 파일을 배치한다.
- 각자 하루에 한 번 이상 의견을 낸다.

**Comment by gw-lim** [comment]
# 3팀

<img width="658" alt="스크린샷 2024-02-26 오후 4 15 45" src="https://github.com/codeit-bootcamp-frontend/1-Weekly-Mission/assets/83965026/43821e33-f7d1-4048-81d7-2502fcba6149">

**Comment by ayden94** [comment]
<img width="2505" alt="스크린샷 2024-02-26 오후 9 57 41" src="https://github.com/codeit-bootcamp-frontend/1-Weekly-Mission/assets/144667387/1e2292bb-bcf5-4372-a3cf-777db202004d">
